static_files/main.js

```diff
@@ -0,0 +1,22 @@
+
+const  Menu = document.getElementById("HiddenMenu");
```

Comment:
Why did you prefer to get the element using `getElementById` instead `querySelector`, that you made in line `5`?